packages/slice-machine/src/modules/userContext/index.ts

```diff
@@ -85,7 +92,32 @@ export const userContextReducer: Reducer<
         hasSeenTutorialsTooTip: true,
       };
     }
+    case getType(refreshStateCreator): {
+      return {
+        ...state,
+        authStatus: getAuthStatus(action.payload.clientError),
+      };
+    }
     default:
       return state;
   }
 };
+
+const getAuthStatus = (
+  clientError: ErrorWithStatus | undefined
+): AuthStatus => {
+  switch (clientError?.status) {
+    case undefined: {
+      return AuthStatus.AUTHORIZED;
+    }
+    case 403: {
+      return AuthStatus.UNAUTHORIZED;
+    }
+    case 401: {
+      return AuthStatus.FORBIDDEN;
```

Comment:
so the usage of 401 and 403 is swapped all around the app. The backend returns a 403 for unauthorised requests, and a 401 for forbidden ones. 
These states are also incorrectly (or I guess correctly to make it work ahah) in the frontend to display the right messages: 
<img width="497" alt="image" src="https://user-images.githubusercontent.com/69144671/184619353-55df087f-4668-4bef-b66a-17b42de5ca15.png">
<img width="607" alt="image" src="https://user-images.githubusercontent.com/69144671/184619396-3a8485f1-fa9a-4c4f-ba76-7531c23c67ec.png">

I think we should refactor the codebase to swap them but I think we should do so in a separate refactoring ticket (since it involves changing code in multiple other places as well as changing the wroom api responses). So for the time being I kept the consistent usage of 403 being unauthorised

Reply:
Cool add it to those notes to the tech debt board :)

Reply:
done :)